workspaces/cli-client/src/client.ts

```diff
@@ -10,15 +10,13 @@ class Client {
     return JsonHttpClient.getJsonWithoutHandlingResponse(url);
   }
 
-  setIdentity(user: IUser | null) {
-    if (user) {
-      const url = `${this.baseUrl}/identity`;
-      return JsonHttpClient.putJson(url, { user });
-    }
+  postTrackingEvents(events: TrackingEventBase<any>[]) {
+    const url = `${this.baseUrl}/tracking/events/apiname`;
+    return JsonHttpClient.postJson(url, { events });
   }
 
-  postTrackingEvents(apiName: string, events: TrackingEventBase<any>[]) {
-    const url = `${this.baseUrl}/tracking/events`;
+  postTrackingEventsWithApi(apiName: string, events: TrackingEventBase<any>[]) {
```

Comment:
This is a little odd, we might want to change the interfaces of these such that you can pass arbitraty data and that gets forwarded through, or it's attached to each event.

Right now this should be identical to what we are currently sending